Week-1/Assignment 2/styles.css

```diff
@@ -0,0 +1,113 @@
+html{
+    height: 100%;
+}
+body{
+    margin:0px;
+    font-family: 'Lato', sans-serif;
+    font-weight: 400;
+    position: relative;
+}
+
+/* Header Specific CSS */
+.mah-content-wrapper{
+    display: flex;
+    background-color: #666161;
+    padding: 24px 100px;
+    color: #fff;
+}
+.mah-meta-data {
+    padding: 0px 35px;
+    position: relative;
+}
+.mah-meta-data h1.mah-title {
+    margin: 10px 0px;
+}
+.mah-meta-data span {
+    display: block;
+    margin-right: 23px;
+}
+.mah-cover-actions {
+    display: flex;
+    position: absolute;
+    bottom: 0px;
+}
+
+/* Footer Specific CSS */
+.music-album-footer{
+    bottom: 0;
+    position: absolute;
+    width: 100%;
+}
+.maf-content-wrapper{
+    display: flex;
+    padding: 9px;
+    color: #fff;
+    background-color: #7066FB;
+}
+.maf-cover-image{
+    width: 4em;
+}
+.maf-cover-image img{
+    width: 100%;
+}
+.maf-meta-data {
+    display: flex;
+    width: 100%;
+}
+.song-info, .maf-cover-actions{
+    flex: 50%;
+}
+.song-info span{
+    display: block;
+    margin: 7px;
+}
+.maf-cover-actions ul {
+    list-style: none;
+    display: flex;
+}
+.maf-cover-actions ul li {
+    flex: 1;
+}
+
+/* Sidebar Specific CSS */
+.sidebar{
+    position: absolute;
+    top: 0;
+    height: 100%;
+    background-color: #eee;
+}
+.sidebar-nav-menu ul{
+    list-style: none;
+    padding: 0px;
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+}
+.sidebar-nav-menu ul li {
+    padding: 1px 24px;
+    line-height: 10vh;
+}
+
+/* Main Content Specific CSS */
+.main-content {
+    margin-left: 4em;
+    padding: 20px;
+}
+.main-content table{
+    width: 100%;
+    border-collapse:collapse;
+}
+.main-content table tr td {
+    text-align: left;
+    padding: 15px;
+}
+.main-content table tr:nth-child(even) {
+    background-color: #f2f2f2;
+}
+.main-content table tr td a {
+    padding: 4px;
+    border: 1px solid;
+    text-transform: uppercase;
+    text-decoration: none;
+    color:#0000ff;
+}
```

Comment:
This is a lint issue. 
There are certain coding standards/practices we follow.
Adding a new line at the end of the file is one such practice.
Please fix this and read about eslint and how to configure a linter in vsCode or any code editor of your choice.